python/beeai_framework/backend/chat.py

```diff
@@ -42,7 +42,7 @@ class ChatModelParameters(BaseModel):
     max_tokens: int | None = None
     top_p: int | None = None
     frequency_penalty: int | None = None
-    temperature: int = 0
+    temperature: int | None = None
```

Comment:
this appears to be undoing: https://github.com/i-am-bee/beeai-framework/pull/457/files#diff-08961e9238c2d441d0988309401935a99763661e09fe1a4a92fcec72e89ac6eaL45-R45